### PR TITLE
Temporary Authentication Page

### PR DESCRIPTION
**Pull request by Mifaki**
Changes Made: 
- Moving all of color and typography palette for ease of use using tailwind `@layer utilities` for tailwind breakpoints compatibility
- Installing `react-hook-form` and `zod` library for faster and more structure form and form validation
- Create Temporary Login and Register page without any server side action

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -8,7 +8,7 @@ APP_NAME=UBTV
 APP_ENV=local
 APP_KEY=
 APP_DEBUG=true
-APP_URL=http://localhost
+APP_URL=http://localhost:8000
```
Comment (mirzahilmi) [inline .env.example:11]: is this changes necessary? may i know the reason why you changed it in the first place

Comment (Mifaki) [reply .env.example:11]: I updated the APP_URL to http://localhost:8000 in the .env.example for simplicity and to match the default Laravel development server URL. This can help avoid potential issues related to CORS or URL mismatches during local development.

Comment (mirzahilmi) [reply .env.example:11]: Okay, that make sense. Thanks for the reply

**Review by mirzahilmi — APPROVED** [review]
LGTM
